tests/html/about.html

```diff
@@ -0,0 +1,186 @@
+<html>
```

Comment:
Sure,

Really I just pulled the bootstrap template body and added my own HTML inside of the pages to get some HTML.  Really, in the grand scheme of things, it doesn't matter, but, since it seems you feel strongly about it, it's just a few lines of code to change.